_includes/partners/contact.html

```diff
@@ -0,0 +1,35 @@
+<main id="main-content">
+  <section>
+    <div class="container partners-container">
+      <div class="grid-row">
+        <div class="tablet:grid-col-9 accordion-container">
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/shell.js"></script>
+          <script>
+            hbspt.forms.create({
+              region: "na1",
+              portalId: "5531666",
+              formId: "e83228a3-efab-4d4d-b16c-d5b531286406"
```

Comment:
WDYT of moving these to config so we can override them with test values for sandbox if needed? similar to how we do the contact form salesforce IDs?

```suggestion
              portalId: "{{ site.partner_portal_id }}",
              formId: "{{ site.partner_form_id }}"
```

Reply:
That's a good idea, one I questioned as well. Valid and noted.